.github/workflows/website.yml

```diff
@@ -24,11 +24,11 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
       with:
-        path: 'aws'
+        path: 'source'
```

Comment:
Now then main repo will be checked out in `/home/runner/work/aws/aws/source` instead of `/home/runner/work/aws/aws/aws`.